actuator/src/main/java/org/tron/core/vm/OpCode.java

```diff
@@ -577,6 +586,19 @@
 
   ISCONTRACT(0xd4, 1, 1, OpCode.Tier.ExtTier),
 
+  STAKE(0xd5, 2, 1, OpCode.Tier.ExtTier),
+
+  UNSTAKE(0xd6, 0, 1, OpCode.Tier.ExtTier),
+
+  WITHDRAWREWARD(0xd7, 1, 1, OpCode.Tier.ExtTier),
+
+  REWARDBALANCE(0xd8, 1, 1, OpCode.Tier.ExtTier),
+
+  ISSRCANDIDATE(0xd9, 1, 1, OpCode.Tier.ExtTier),
+
+  TOKENISSUE(0xda, 1, 1, Tier.HighTier),
```

Comment:
Since all other OpCodes are all verb-phrase, should this be `ISSUETOKEN`?

Reply:
Will consider to rename the opcode in next release due to code freezing.

Reply:
I know that if it is released, then It will never be changed.
So sad that your release plan is more important that code and consistency.

Reply:
It can be changed. Not a hard fork case.